CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Was this empty line added by mistake?

Reply:
Oh, it's a mistake, I was thinking if I should put the options at the head of the file like [googletest does](https://github.com/google/googletest/blob/main/googletest/CMakeLists.txt#L13) so that we won't neglect their existence, but I think it's OK for them to stay where they are now. The empty line will be removed